tests/Hosting.Services.Web.UnitTests/Middlewares/UserIdentity/UserIdentiyMiddlewareTests.cs

```diff
@@ -57,6 +57,29 @@ public async Task CreateUserHash_UseSalt()
 			Assert.AreNotEqual(changedHash, initialHash);
 		}
 
+		[TestMethod]
+		public async Task CreateUserHash_UseStaticSalt()
+		{
+			HttpContext context = HttpContextHelper.GetContextWithEmail("Abc123@outlook.com");
+
+			Random random = new();
+			byte[] saltValue = new byte[128];
+
+			TestIdentityProvider provider = new ("ProviderWrapper", new EmailBasedUserIdentityProvider());
```

Comment:
When constructing `EmailBasedUserIdentityProvider`, add a `using` block to ensure it is properly disposed

Reply:
Do the same with `TestStaticSaltProvider`, although there is currently no deleting logic, in case someone adds it in future

Reply:
Done with SaltProviders. For IdentityProviders, it is not possible to use "using" block as they don't have dispose method